.env.example

```diff
@@ -3,30 +3,16 @@
 #=============================================
 ALCHEMY_API_KEY=
 ETH_NETWORK=sepolia
-# IMX API endpoint
-PUBLIC_API_URL=https://api.sandbox.x.immutable.com/v1
-# sandbox contracts in sepolia
-STARK_CONTRACT_ADDRESS=0x2d5C349fD8464DA06a3f90b4B0E9195F3d1b7F98
-REGISTRATION_ADDRESS=0xDbA6129C02E69405622fAdc3d5A7f8d23eac3b97
-GAS_LIMIT=7000000
-GAS_PRICE=40000000000
 
-# Your environment's api key generated from https://hub.immutable.com/
-API_KEY=
-#=============================================
-# Bulk Minting
-#=============================================
-PRIVATE_KEY1=
-TOKEN_ID=
-TOKEN_ADDRESS=
-BULK_MINT_MAX=50
 
 #=============================================
 # Onboarding
 #=============================================
+# Your environment's api key generated from https://hub.immutable.com/
+API_KEY=
 OWNER_ACCOUNT_PRIVATE_KEY=
-COLLECTION_CONTRACT_ADDRESS=
 COLLECTION_PROJECT_ID=
```

Comment:
Just to be clear, this is the legacy project id in Hub right? I think it's better to explain a comment given the name change.